test/language/expressions/arrow-function/arrow/arrayfunctions_3.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: arrow function array manipulations
+es6id: 14.2
+---*/
+
+var arr = [1,2,3];
+ var result = 0;
+
+ arr.forEach(v => {
+            result += v;
+ });
+
+assert.sameValue(result, 6, "the result value should be 6.");

```

Comment:
Duplicates

- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/cannot-override-this-with-thisArg.js